plugins/foreman_templates/5.0/index.md

```diff
@@ -84,6 +84,31 @@ After the installation there are new settings available. These are the default s
 
 Settings above are preconfigured for importing from upstream community-templates repository.
 
+## 3.1 Example of configuration github repository
+
+Let's assume you have a github account with login `template_master`. Here are the steps how to configure foreman_templates so that you can import and export templates between Foreman and your github repository. Let's assume the repository is called `templaes` and it has branch called `master`.
+
+1. First we'll configure settings. Set a `branch` setting to `master`, `prefix` setting to empty string and `repo` to `ssh://git@github.com/template_master/templates`.
+2. We'll need to import github SSH public key to foreman user, run `sudo -u foreman ssh github.com`, verify and save the key. You'll probably see permission denied message since the ssh connection will not succeed but that's expected.
```

Comment:
Probably better to call this the `GitHub SSH host key` so as not to confuse with the public part of the personal key.